### ISSUE TITLE
SpiderMonkey: process.memoryUsage

### ISSUE THREAD

**Issue by martinies** [jxcore/jxcore#200]
On SpiderMonkey build, process.memoryUsage returns 0 for JS engine memory stats.

**Comment by ktrzeciaknubisa** [comment]
Yes, that's right. Thanks for reminding. Apparently this was left as TODO mark. We still have left few such things though. We're trying to approach them one by one gradually.
